functions.php

```
@@ -1104,3 +1125,446 @@ function load_more_parent_cat_news()
+            return false;
+        }
+    }
+
+    return $show;
+}
+
+// Add direct logout tab to BuddyPress profile navigation
+add_action('bp_setup_nav', 'add_direct_logout_tab_to_buddypress_profile', 100);
+function add_direct_logout_tab_to_buddypress_profile()
+{
+    global $bp;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$bp'. (undefined)

(UnusedLocalVariable)